src/main/java/uk/gov/hmcts/juror/api/moj/service/PoolCreateServiceImpl.java

```diff
@@ -513,7 +518,7 @@ private JurorPool createJurorPool(String login, String owner, Voters voter,
 
         jurorPool.setJuror(juror);
 
-        jurorPoolRepository.saveAndFlush(jurorPool);
+        //jurorPoolRepository.save(jurorPool);
```

Comment:
Why do we no longer save the juror here?

Reply:
This is because the save is done later after we have created the new entity. Removed comment.